ThePhasesOfCraftship/2_best_practice_first/strategicDesignPart1/exercises/1_RefactoringTo4Tiers/begin/tests/e2e/createAssignment.test.ts

```diff
@@ -0,0 +1,91 @@
+import { defineFeature, loadFeature } from "jest-cucumber";
+import path from "path";
+import supertest from "supertest";
+import request from "supertest";
+import { app, Errors } from "../../src";
+import { resetDatabase, ClassRoomBuilder } from "../fixtures";
+
+const feature = loadFeature(
+  path.join(__dirname, "../features/createAssignment.feature")
+);
+
+defineFeature(feature, (test) => {
+
+    beforeEach(resetDatabase);
```

Comment:
I also didn't know about this trick haha. So I guess Jest runner automatically calls it by just passing the function reference?

Reply:
Yes, it's basically a shorthand for 
```
beforeEach(async () => await resetDatabase())
```